cypress/e2e/demoqa.cy.js

```diff
@@ -0,0 +1,40 @@
+describe('demoQA - Registion Form', () => {
+  const testInput = {
+    firstName: 'Nghia',
+    lastName: 'Do',
+    email: 'meomeo211@gmail.com',
+    gender: 'Female',
+    phoneNumber: '0774859984',
+    //DOB: '14/11/2001',
+    subjects: 'Economics',
+    hobby: '2',
+    address: 'Ho Chi Minh City',
+    state: 'NCR',
+    city: 'Delhi',
+  }
+    it('User sucessfully submit', () => {
```

Comment:
There are 2 positive cases for this form
- Submit with all fields
- Submit with mandatory fields only